src/cohort.js

```diff
@@ -0,0 +1,205 @@
+const Twilio = require('../src/sms.js')
+class Cohorts {
+  constructor() {
+    this.cohortList = []
```

Comment:
It looks like you are only using one class and putting everything in this array as objects?